src/bin/compute-wer.cc

```diff
@@ -91,36 +119,26 @@ int main(int argc, char *argv[]) {
       }
       num_words += ref_sent.size();
       int32 ins, del, sub;
-      word_errs += LevenshteinEditDistance(ref_sent, hyp_sent, &ins, &del, &sub);
+      int32 this_word_errs = LevenshteinEditDistance(ref_sent, hyp_sent, &ins, &del, &sub);
+      word_errs += this_word_errs;
       num_ins += ins;
       num_del += del;
       num_sub += sub;
 
       num_sent++;
-      sent_errs += (ref_sent != hyp_sent);
+      int32 this_sent_errs = (ref_sent != hyp_sent);
+      sent_errs += this_sent_errs;
+
+      if (GetVerboseLevel() > 1)
+        PrintWerResults(std::cerr, ref_sent.size(), 1, this_word_errs, this_sent_errs, ins, del, sub, key + " ", 0);
```

Comment:
I'm not crazy about the way you are doing this.  In general, parsing things from log files is not the way things are done in Kaldi. I think it would be preferable to add an option --detailed-error-output (accepting a wxfilename), to which would be written a text file of the format:
  utt1  ref ins del sub
  utt2  ref ins del sub
and so on.